admin/fluent-bit/Makefile

```diff
@@ -0,0 +1,47 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fluent-bit
+PKG_VERSION:=3.1.0
+PKG_RELEASE:=1
+
+PKG_SOURCE=v$(PKG_VERSION).tar.gz
```

Comment:
This is not correct. We can not download files "vVERSION", because in the dl folder it is going to be mess and we would not be sure to which package does it belong, yeah, you can verify it manually, but you probably won't verify it, if it was not overwritten by the other package with the same version. 

So, it should be as usual as it is done in other Makefiles - packageName-Version.

Reply:
done